packages/auto-instrumentations-node/src/instrumentations/extentions/http.ts

```diff
@@ -106,12 +106,9 @@ function createHttpRequestHook(
         //rollback 'write()' only after the end() function is called.
         clientRequest.write = originalWrite;
         clientRequest.end = originalEnd;
+        // add the 'body' in case end(body) was called
+        bodyHandler.addChunk(_args[0]);
```

Comment:
is there a try/catch somewhere along the way? (we just had an issue with python in which chunk couldn't be decoded)

Reply:
I added try-catch to 'addChunk'.